.github/test-spec.yml

```diff
@@ -21,7 +21,13 @@
   - "subsys/testsuite/ztest/**/*"
 
 "CI-lwm2m-test":
```

Comment:
As we have checked, in the lwm2m carrierlib team, it seems we might not need any test run on zephyr-sdk modifications. Would it be possible to leave this as blank? or remove the label entirely?

Reply:
yes just remove line 24, and it won't be run

Reply:
That worked!